scripts/download_json/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+/**
+ * Random scripts
+ * @module Scripts
+ */
+
+/**
+ * Generates the string to use as download link of a object in JSON format
+ *
+ *
+ * @param {object} Object - Object that will be downloaded as JSON
+ * Example:
+ *  <a href={getJSONDownloadString(object)} download={`${fileName}.json`}>
```

Comment:
To write an example doc use the `@example` [tag](http://usejsdoc.org/tags-example.html).
Also, to include this new method into scripts module, you might want to use the `@memberof module:Scripts` instead of `@module Scripts`